crates/cargo-codspeed/src/build.rs

```diff
@@ -18,9 +18,14 @@ fn get_compile_options(
     features: &Option<Vec<String>>,
     package: &Package,
     benches: Vec<&str>,
+    is_root: bool,
```

Comment:
```suggestion
    is_root_package: bool,
```
This would be more obvious